docs/sdks/uid2-sdk-ref-java.md

```diff
@@ -63,56 +58,165 @@ The binary is published on the Maven repository:
 
 - [https://central.sonatype.com/artifact/com.uid2/uid2-client](https://central.sonatype.com/artifact/com.uid2/uid2-client)
 
-## Usage for DSPs
+## Initialization
+
+The initialization step depends on the role, as shown in the following table.
+
+| Role        | Create Instance of Class | Link to Instructions                              |
+|:------------| :--- |:--------------------------------------------------|
+| `DSP`       | BidstreamClient | [Usage for DSPs](#usage-for-dsps)                 |
+| `Sharer`    | SharingClient | [Usage for UID2 Sharers](#usage-for-uid2-sharers) |
+| `Publisher` | Uid2PublisherClient | [Usage for Publishers](#usage-for-publishers)     |
 
-The initialization function configures the parameters necessary for the SDK to authenticate with the UID2 service. It also allows you to configure retry intervals in the event of errors.
+You will need to provide the values necessary for the SDK to authenticate with the UID2 service.
 
-| Parameter | Description | Recommended Value |
-| :--- | :--- | :--- |
-| `endpoint` | The endpoint for the UID2 service. | N/A |
-| `authKey` | The authentication token that belongs to the client. For access to UID2, see [Contact Info](../getting-started/gs-account-setup.md#contact-info). | N/A |
+| Parameter | Description | 
+| :--- | :--- |
+| `endpoint` | The endpoint for the UID2 service. See [Environments](../getting-started/gs-environments) | 
```

Comment:
We have a mix of names across the different clients :( This is called `baseUrl` for `BidstreamClient` and `SharingClient` and `uid2BaseUrl` for `PublisherUid2Client`

Reply:
Yeah, that makes sense.